src/Halcyon/MemoryRepository.php

```diff
@@ -47,18 +47,18 @@ public function get($key, $default = null)
      *
      * @param  string  $key
      * @param  mixed   $value
-     * @param  \DateTimeInterface|\DateInterval|float|int  $minutes
+     * @param  \DateTimeInterface|\DateInterval|int  $seconds
      * @return void
      */
-    public function put($key, $value, $minutes = null)
+    public function put($key, $value, $seconds = null)
```

Comment:
Is there anywhere we need to ensure is updated to pass seconds instead of minutes to this?

Reply:
@LukeTowers very likely, but that's still a task to complete in the list.